test/08_manylinuxXXXX_only/cibuildwheel_test.py

```diff
@@ -12,9 +14,11 @@ def test():
     # rather than when dynamically loading the Python
     actual_wheels = utils.cibuildwheel_run(project_dir, add_env={
         'CIBW_ENVIRONMENT': 'CFLAGS="$CFLAGS -Werror=implicit-function-declaration"',
+        'CIBW_SKIP': 'cp27*',  # not available on manylinux2014
+        'CIBW_MANYLINUX_X86_64_IMAGE': manylinux_image,
+        'CIBW_MANYLINUX_I686_IMAGE': manylinux_image,
     })
 
-    # also check that we got the right wheels
-    expected_wheels = [w for w in utils.expected_wheels('spam', '0.1.0')
-                       if not '-manylinux' in w or '-manylinux2010' in w]
+    expected_wheels = [w.replace('manylinux2010', manylinux_image) for w in utils.expected_wheels('spam', '0.1.0')
+                       if '-cp27' not in w and '-manylinux2010' in w]
```

Comment:
Meh. Is there nothing better we can do? I already didn't like the previous lines.

`if '-cp27' not in w and '-{}_'.format(manylinux_image) in w` ?

Reply:
> if '-cp27' not in w and '-{}_'.format(manylinux_image) in w

Won't work with `manylinux2014`

If you find something easier, feel free to commit.

Reply:
Why so?

Actually, now realizing: you're not updating the list of `utils.expected_wheels`, because it's not our default yet.
Maybe we should add an extra argument to `utils.expected_wheels` to select the right `manylinux` version?

Reply:
When I saw `utils.expected_wheels` while refactoring tests, I indeed thought that it could be improved. Shall be in another PR though don't you think ?

Reply:
Hmmm, for refactoring, yes. But the main thing I'm noticing for this PR is that we're breaking the current rule/habit that all the wheel names are to be found in `utils.expected_wheels`, and that we filter out the ones that are not expected in a test.

Reply:
I've hacked something together to get the pattern 'only filter out'

Reply:
Nice! I've wanted to do something like this before, but I never was happy with the way to handle the 2-phase format-strings, but ... this could work. I should see how #143 gets rebased onto this but ... :-)

Just a suggestion; up to you to judge whether you like it or not: what about only keeping the loop over `manylinux_images`, and keep the listing of different Python versions and architectures explicit? This would keep consistency with the other platforms, and keep a nice mapping to the build identifiers. But again, just a suggestion. Up to your judgment.

Reply:
I finally kept consistency by rewriting the way identifiers are built on all architectures using PEP 425 as a baseline.

Reply:
Nice! :-)